src/framework/core/js/DataBinding.js

```diff
@@ -853,6 +853,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     fluid.establishModelRelay = function (that, optionsModel, optionsML, optionsMR, applier) {
+        var shadow = fluid.shadowForComponent(that);
```

Comment:
Shall a test be added to cover the case that model relay is not merged/established more than once?